### PR TITLE
ActivityPub design fixes and improvements

### PR DESCRIPTION
**Pull request by djordjevlais**

### CODE REVIEW

File: apps/admin-x-activitypub/src/components/Activities.tsx [hunk 1]
```diff
@@ -91,7 +91,8 @@ const Activities: React.FC<ActivitiesProps> = ({}) => {
         data,
         fetchNextPage,
         hasNextPage,
-        isFetchingNextPage
+        isFetchingNextPage,
+        isLoading // Add this to destructure isLoading from the hook
```
Comment (mike182uk) [inline apps/admin-x-activitypub/src/components/Activities.tsx:95]:
```suggestion
        isLoading
```

Comment not needed 👍

File: apps/admin-x-activitypub/src/components/Activities.tsx [hunk 2]
```diff
@@ -139,14 +140,17 @@ const Activities: React.FC<ActivitiesProps> = ({}) => {
         <>
             <MainNavigation title='Activities' />
             <div className='z-0 flex w-full flex-col items-center'>
-                {activities.length === 0 && (
+                {isLoading ? (
+                    <div className='mt-8 flex flex-col items-center justify-center space-y-4 text-center'>
+                        <LoadingIndicator size='lg' />
+                    </div>
+                ) : activities.length === 0 ? (
```
Comment (mike182uk) [inline apps/admin-x-activitypub/src/components/Activities.tsx:147]: I'm not a fan of nested ternaries, they feel hard to reason about / hard to read. What do you think about doing this in "blocks" instead?

Something like:

```
{
	isLoading && (
	  <div className='mt-8 flex flex-col items-center justify-center space-y-4 text-center'>
	      <LoadingIndicator size='lg' />
	  </div>
	)
}
{
	isLoading === false && activities.length === 0 ? (
		<div className='mt-8'>
            <NoValueLabel icon='bell'>
                When other Fediverse users interact with you, you&apos;ll see it here.
            </NoValueLabel>
        </div>
	)
}
{
	(isLoading === false && activities.length > 0) ? (
		...
	)
}
```

File: apps/admin-x-activitypub/src/components/Search.tsx [hunk 3]
```diff
@@ -141,7 +141,7 @@ const Search: React.FC<SearchProps> = ({}) => {
                 {showLoading && (
                     <LoadingIndicator size='lg'/>
                 )}
-                {showNoResults && (
+                {showNoResults && (query.length > 0) && (
```
Comment (mike182uk) [inline apps/admin-x-activitypub/src/components/Search.tsx:144]: We could move this extra conditional into the `showNoResults` variable to keep it concise

How come you needed to add `query.length > 0` anyways?

File: apps/admin-x-activitypub/src/components/Activities.tsx (hunk 2 above)
Comment (djordjevlais) [reply apps/admin-x-activitypub/src/components/Activities.tsx:147]: @mike182uk yeah this looks easier to understand, I’ll change it 🫡

**Pull request merged**